sage-android/Sage/app/src/main/java/blueprint/com/sage/admin/browse/fragments/SchoolFormAbstractFragment.java

```diff
@@ -194,11 +194,47 @@ public void onMapReady(GoogleMap map) {
             latLng = new LatLng(mSchool.getLat(), mSchool.getLng());
         }
         moveMapToLatLng(latLng);
+        mBounds = latLng;
+
+        mRadius.setMax(MAX_RADIUS);
+        final int initialRadius;
+        if (mSchool != null) {
+            initialRadius = mSchool.getRadius();
+            mRadius.setProgress(initialRadius);
+        } else {
+            initialRadius = 0;
+        }
+
+        mRadiusInt.setText(Integer.toString(initialRadius) + " m");
```

Comment:
String.valueOf is much better
Come to think of it, you might not even need it because int + string = string i believe.